### DISCUSSION TITLE
Is v3 ready for use

### DISCUSSION THREAD

**Discussion by tolahs** [elsa-workflows/elsa-core#3786] [General]
Hi I am new for elsa. it is looking amazing . We are planing to adapt Elsa to our project. But I want to be sure v3 is ready for production

**Comment by dodyg** [comment]
The documentation isn't ready yet.

**Reply by dodyg** [reply]
I am learning by making samples here https://github.com/elsa-workflows/elsa-core/discussions/3696.

**Comment by sfmskywalker** [comment]
It’ll be at least one month, maybe two months, before we do a release candidate, and then another month for release. We want the documentation to be ready as well (although that will be an ongoing effort).

**Reply by tolahs** [reply]
thanks for reply and your hard work.
